server/src/main/java/org/opensearch/common/cache/ICache.java

```diff
@@ -31,4 +33,8 @@ public interface ICache<K, V> {
     long count();
 
     void refresh();
+
+    void close();
```

Comment:
This doesn't seem like a property of a 'cache', lets add or check for Closable or other existing interfaces separately from caching systems.

Reply:
Made ICache extend Closeable instead of using this.